PlayFabClientSDK/source/PlayFabSerializer.cs

```diff
@@ -0,0 +1,28 @@
+using PlayFab.Json;
+
+namespace PlayFab
+{
+    public class PlayFabSerializer: ISerializerPlugin
+    {
+        /// <summary>
+        /// Serializes a data object.
+        /// </summary>
+        /// <param name="object">The data object.</param>
+        /// <returns>Serialized object in string form.</returns>
+        public string SerializeObject<T>(T @object)
+        {
+            return JsonWrapper.SerializeObject(@object);
```

Comment:
JsonWrapper is in effect, this exact type of wrapper.
There's no reason to re-wrap it again.
I'd replace JsonWrapper with two things that implement ISerializerPlugin.